src/sentry/sentry_metrics/consumers/indexer/processing.py

```diff
@@ -26,6 +26,14 @@ def __init__(self, config: MetricsIngestConfiguration):
         self._indexer = STORAGE_TO_INDEXER[config.db_backend](**config.db_backend_options)
         self._config = config
 
+    def __getstate__(self) -> MetricsIngestConfiguration:
```

Comment:
Mind adding a comment to explain why we override `__getstate__` and `__setstate__`? It is not evident if you do not know about this PR